index.html

```diff
@@ -0,0 +1,89 @@
+    <link rel="stylesheet" href="assets/index.css" />
+    <link
+      rel="shortcut icon"
+      href="https://upload.wikimedia.org/wikipedia/commons/thumb/f/fa/Apple_logo_black.svg/505px-Apple_logo_black.svg.png"
+      type="image/x-icon"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200&display=swap"
+      rel="stylesheet"
+    />
+    <title>Apple Clone</title>
+  </head>
+  <body>
+    <main class="container">
+      <nav class="navigation">
+        <ul class="nav-list">
+          <li class="link">
+            <a href="#">
+              <img src="assets/img/logo.png" alt="Apple-logo" class="logo"
+            /></a>
+          </li>
+          <li class="link"><a href="#">Store</a></li>
+          <li class="link"><a href="#">Mac</a></li>
+          <li class="link"><a href="#">iPod</a></li>
+          <li class="link"><a href="#">iPhone</a></li>
+          <li class="link"><a href="#">iPad</a></li>
+          <li class="link"><a href="#">iTunes</a></li>
+          <li class="link"><a href="#">Support</a></li>
+        </ul>
+        <form action="search" class="search">
+          <input type="text" placeholder="" class="input" />
+        </form>
+      </nav>
+      <section class="hero">
+        <div class="heading">
+          <h1 class="header1">What will your verse be?</h1>
+          <p>
+            Everyone has something to share. See how people are using iPad in
+            there unique way to add to the world's story
+          </p>
+          <p class="para">
+            <a href="#" class="anchor">
+              Watch the verse film and explore the stories.
+              <img
+                src="./assets/img/play-btn.png"
+                alt="play button"
+                class="play-btn"
+              />
+            </a>
+          </p>
+        </div>
+      </section>
+      <div class="grid">
+        <div class="grid-1"></div>
+        <div class="grid-2"></div>
+        <div class="grid-3"></div>
+        <div class="grid-4"></div>
+      </div>
+    </main>
+
+    <footer class="footer">
+      <div class="main-footer">
+        <span>
+          Shop the <a href="#">Apple Online Store</a> (1-800-MY-APPLE), visit an
+          <a href="#">Apple Retail Store</a>, or find a
+          <a href="#">reseller</a>.
+        </span>
+        <ul class="footer-links">
+          <li><a href="#">Site Map</a></li>
+          <li><a href="#">Hot News</a></li>
+          <li><a href="#">RSS Feeds</a></li>
+          <li><a href="#">Media Info</a></li>
+          <li><a href="#">Environment</a></li>
+          <li><a href="#">Job Opportunities</a></li>
+          <li><a href="#">Contact Us</a></li>
+        </ul>
+      </div>
+      <span class="contact">
+        Copyright © 2014 Apple Inc. All rights reserved.
+        <a href="#">Terms of Use </a> <a href="#">Privacy</a> Policy</span
+      >
+    </footer>
```

Comment:
- [ ] If you could find a font closer to the original page that would be great! Also, please align the text in the left so they both start at the same point.

> ![image](https://user-images.githubusercontent.com/57414789/82129804-1cf09a80-978b-11ea-971e-ef78a2011b1d.png)